microcloud/service/lxd_config.go

```diff
@@ -0,0 +1,180 @@
+package service
+
+import (
+	"fmt"
+	"net"
+	"strconv"
+
+	"github.com/canonical/lxd/shared/api"
+)
+
+// DefaultPendingFanNetwork returns the default Ubuntu Fan network configuration when
+// creating a pending network on a specific cluster member target.
+func (s LXDService) DefaultPendingFanNetwork() api.NetworksPost {
+	return api.NetworksPost{Name: "lxdfan0", Type: "bridge"}
+}
+
+// DefaultFanNetwork returns the default Ubuntu Fan network configuration when
+// creating the finalized network.
+func (s LXDService) DefaultFanNetwork() (api.NetworksPost, error) {
+	underlay, _, err := defaultGatewaySubnetV4()
+	if err != nil {
+		return api.NetworksPost{}, fmt.Errorf("Couldn't determine Fan overlay subnet: %w", err)
```

Comment:
I tend to avoid using contractions in user messages, i.e "Couldn't" I would tend to use use "Failed to" or "Could not".

@ru-fu is this something you avoid in the docs too out of interest?

Reply:
Most style guides advise to avoid contractions, but the Ubuntu style guide actually encourages them: https://docs.ubuntu.com/styleguide/en#contractions
I use them occasionally, but I'm way too used to NOT using them, so it's rather rare. ;)